src/aspire/basis/__init__.py

```diff
@@ -1,11 +1,19 @@
+# We'll tell isort not to sort these base classes
+# isort: off
+
 from .basis import Basis
```

Comment:
Not sure what you mean here.  The sorting is needed because the fb_* classes need the steerable basis first... and steerable needs basis first (so it can't be in the later block).

Reply:
Nm I just didn't understand circular imports. I see now.